metricbeat/module/postgresql/_meta/kibana/7/dashboard/Metricbeat-Postgresql-overview.json

```diff
@@ -0,0 +1,737 @@
+              "type": "top_hits"
+            },
+            {
+              "enabled": true,
+              "id": "2",
+              "params": {
+                "field": "postgresql.activity.database.name",
+                "missingBucket": false,
+                "missingBucketLabel": "Missing",
+                "order": "desc",
+                "orderBy": "_key",
+                "otherBucket": false,
+                "otherBucketLabel": "Other",
+                "size": 5
+              },
+              "schema": "bucket",
+              "type": "terms"
+            },
+            {
+              "enabled": true,
+              "id": "3",
+              "params": {
+                "field": "postgresql.activity.user.name",
+                "missingBucket": false,
+                "missingBucketLabel": "Missing",
+                "order": "desc",
+                "orderBy": "_key",
+                "otherBucket": false,
+                "otherBucketLabel": "Other",
+                "size": 5
+              },
+              "schema": "bucket",
+              "type": "terms"
+            },
+            {
+              "enabled": true,
+              "id": "4",
+              "params": {
+                "field": "postgresql.activity.application_name",
+                "missingBucket": false,
+                "missingBucketLabel": "Missing",
+                "order": "desc",
+                "orderBy": "_key",
+                "otherBucket": false,
+                "otherBucketLabel": "Other",
+                "size": 5
+              },
+              "schema": "bucket",
+              "type": "terms"
+            }
+          ],
+          "params": {
+            "perPage": 10,
+            "showMetricsAtAllLevels": false,
+            "showPartialRows": false,
+            "showTotal": false,
+            "sort": {
+              "columnIndex": null,
+              "direction": null
+            },
+            "totalFunc": "sum"
+          },
+          "title": "[Postgresql] Server Activity ",
+          "type": "table"
+        }
+      },
+      "id": "508fb7b0-f3bb-11e8-a48e-69084fbf7c82",
+      "type": "visualization",
+      "updated_at": "2018-12-05T14:37:54.101Z",
+      "version": 4
+    },
+    {
+      "attributes": {
+        "description": "",
+        "hits": 0,
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [
+              {
+                "$state": {
+                  "store": "appState"
+                },
+                "meta": {
+                  "alias": null,
+                  "disabled": false,
+                  "index": "metricbeat-*",
+                  "key": "postgresql.activity.query",
+                  "negate": true,
+                  "params": {
+                    "query": "",
+                    "type": "phrase"
+                  },
+                  "type": "phrase",
+                  "value": ""
+                },
```

Comment:
I see in the screenshot that there is a filter for empty queries, when does it happen? Should we use this filter only in the visualization of recent queries?